modules/getting-started-web-console-creating-route.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-web-console.adoc
+
+[id="getting-started-web-console-creating-route_{context}"]
+= Creating a route
+
+External clients can access applications running on {product-title} through the routing layer and the data object behind that is a Route. The default {product-title} router (HAProxy) uses the HTTP header of the incoming request to determine where to proxy the connection.
+
+You can optionally define security, such as TLS, for the Route.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console.
+* You have a deployed image.
+* You are in the *Administrator* perspective.
+* You must have `cluster-admin` privileges.
+
+.Procedure
+
+. From the *Administrator* perspective, click *Networking* -> *Routes*.
+. Click *Create Route*.
+. Enter the following:
+.. Enter `national-parks-app` in the *Name* field.
+.. From the Service field, select *parksmap*.
+.. Select *8080* as the *Target Port*.
```

Comment:
On section of Creating a route -> Procedure -> 3.c 
8080 should be update as '8080->8080(TCP)' which consistent with the text on OCP page